src/qt/optionsmodel.h

```diff
@@ -61,6 +61,7 @@ class OptionsModel : public QAbstractListModel
         Language,               // QString
         UseEmbeddedMonospacedFont, // bool
         CoinControlFeatures,    // bool
+        SubFeeFromAmount,       //bool
```

Comment:
nit, why not follow the surrounding style?
```suggestion
        SubFeeFromAmount,       // bool
```